src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,13 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
-    public Bulldozer() {
+    private String speed;
+    public Bulldozer(String color , String name , String speed) {
```

Comment:
The class is missing a no-args constructor. According to the task description, you should not remove the no-args constructor. Please add a no-args constructor to this class.